kse/src/org/kse/gui/dialogs/extensions/resources_de.properties

```diff
@@ -94,9 +95,17 @@ DExtendedKeyUsage.jcbOcspStamping.text                = OCSP Zeitstempel
 DExtendedKeyUsage.jcbSmartcardLogon.text              = Anmeldung mit Smartcard
 DExtendedKeyUsage.jcbTimeStamping.text                = Zeitstempel
 DExtendedKeyUsage.jcbTlsWebClientAuthentication.text  = Beglaubigung von TLS Web-Clients
-DExtendedKeyUsage.jcbTlsWebServerAuthentication.text  = Beglaubigung von TLS Web-Servern
+DExtendedKeyUsage.jcbTlsWebServerAuthentication.text  = Beglaubigung von TLS Web-Servern
+DExtendedKeyUsage.jcbCustomExtendedKeyUsage.text      =
 DExtendedKeyUsage.jlExtendedKeyUsage.text             = Ausweitung der Schl\u00FCsselbenutzung
 
+DCustomExtendedKeyUsage.Title=
```

Comment:
It is better to not add any keys for languages that you cannot translate yourself to. Then the english phrases are shown instead of nothing. Translation tools like ResourceBundle Editor make it easy to find missing texts.